### PR TITLE
Table changes

### PR DESCRIPTION
**Pull request by JRMBowers**
Implementation of table changes, user story 6 and rename/deletion of database.rb/space.rb. Changes to database table column names.

### CODE REVIEW

File: db/migrations/01_makers_bnb.sql [hunk 1]
```diff
@@ -1 +1 @@
-CREATE TABLE spaces(id SERIAL PRIMARY KEY, name VARCHAR(60)) ;
```
Comment (jjegarcia) [inline db/migrations/01_makers_bnb.sql:1]: can you do space_id, space_name and space_description?

**Review by jjegarcia — CHANGES_REQUESTED** [review]

File: db/migrations/01_makers_bnb.sql (hunk 1 above)
Comment (jjegarcia) [reply db/migrations/01_makers_bnb.sql:1]: also may need to match names to parameters in the code also

File: app.rb [hunk 2]
```diff
@@ -17,12 +18,12 @@ class MakersBnB < Sinatra::Base
   end
 
   post '/add_space' do
-    Database.add_space(name: params[:space_name])
+    Space_handler.add_space(name: params[:name], description: params[:description], price_per_night: params[:price_per_night])
```
Comment (jjegarcia) [inline app.rb:21]: could reane to space_name, space_description

File: model/space_handler.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+class Space_handler
+  attr_reader :id, :name, :description, :price_per_night
+
+  def initialize(id:, name:, description:, price_per_night:)
```
Comment (jjegarcia) [inline model/space_handler.rb:4]: please rename to space_xxxxxx

File: model/space_handler.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    connection.exec_params(
+      "INSERT INTO spaces(spaces_name, spaces_description, price_per_night) VALUES($1, $2, $3);",
+      [name, description, price_per_night],
+    )
+  end
+
+  def self.get_spaces
+    connection = Space_handler.connect
+    result = connection.exec_params('SELECT * FROM spaces;')
+    result.map { |space| Space_handler.new(id: space['spaces_id'], name: space['spaces_name'], description: space['spaces_description'], price_per_night: space['price_per_night']) }
```
Comment (jjegarcia) [inline model/space_handler.rb:30]: please rename to space_xxxxxxx

File: spec/feature/add_space_spec.rb [hunk 5]
```diff
@@ -1,12 +1,15 @@
...
     visit('/add_space')
-    fill_in(:space_name, with: 'Bat Cave')
+    fill_in(:name, with: 'Bat Cave')
+    fill_in(:description, with: 'Its a bat cave')
+    fill_in(:price_per_night, with: 2.55)
     click_on 'Submit'
     result = connect_to_db.exec('SELECT * FROM spaces')
-    expect(result.first['name']).to eq 'Bat Cave'
-  end 
+    expect(result.first['spaces_name']).to eq 'Bat Cave'
```
Comment (jjegarcia) [inline spec/feature/add_space_spec.rb:10]: please rename to space_xxxxxxx

File: views/add_space.erb [hunk 6]
```diff
@@ -1,4 +1,6 @@
 <form action='/add_space' method='post'>
-<input type='text' name='space_name'>
-<input type='submit' value="Submit">
+  <input type='text' name='name'>
```
Comment (jjegarcia) [inline views/add_space.erb:2]: please rename to space_xxxxxxx

**Review by jjegarcia — APPROVED** [review]

**Review by yamlcamel — COMMENTED** [review]
Yeah just those column names everything else looks spot on!

**Review by mbrosch3-ford — APPROVED** [review]

**Pull request merged**